### ISSUE TITLE
cherry fails to copy file

### ISSUE THREAD

**Issue by bernt-matthias** [KennthShang/PhaBOX#38]
```
phabox2 --task cherry  --dbdir '/home/berntm/projects/mb-galaxy-tools/tools/phabox/test-data/phabox_db_v2/' --outpth output/ --contigs '/tmp/tmpj_qx6mfj/files/a/c/9/dataset_ac9084b8-6509-4549-8bac-d09fb8c04a58.dat' --midfolder intermediate/ --len 3000 --threads "${GALAXY_SLOTS:-1}"   --aai 75.0 --share 15.0 --pcov 80.0   --cpident 90.0 --ccov 90.0 --blast blastn
PhaBOX2 is running with: 1 threads!
Running program: CHERRY (Host prediction)
[1/8] filtering the length of contigs...
[2/8] no MAGs provided, skipping MAG CRISPRs prediction...
[3/8] will continue running CHERRY with database bersion...
[4/8] predicting DB CRISPRs...
[5/8] calling genes with prodigal...
[5/8] running all-against-all alignment...
[6/8] generating cherry networks...
100%|██████████████████████████████████████████████████| 390/390 [00:00<00:00, 2115.73it/s]
[6/8] predicting the host...
[7/8] summarizing the results...
[8/8] writing the results...
Command 'cp output//filtered_contigs.fa output//final_prediction//phagcn_supplementary/all_predicted_contigs.fa' failed with exit code 1
cp: cannot create regular file 'output//final_prediction//phagcn_supplementary/all_predicted_contigs.fa': No such file or directory
```

I think the problem is here: https://github.com/KennthShang/PhaBOX/blob/9f34eb0a2d4393268a61c3ed54a6e04b3aa4d922/src/phabox2/cherry.py#L666

Maybe you should have a little flame test in github actions :)

**Comment by KennthShang** [comment]
Thanks. The script is updated. I will try to figure out how to use the actions.

**Comment by KennthShang** [comment]
The issue is fixed in version 2.1.5
